components/sysutils/e2fsprogs/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+#
+# Copyright 2022 Stephan Althaus
+#
+
+BUILD_BITS=		64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		e2fsprogs
+COMPONENT_VERSION=	1.46.4
+#COMPONENT_REVISION=	1
+COMPONENT_SUMMARY=	Second extended file system (ext2fs) management programs
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= \
+	sha256:b11042533c1b1dcf17512f0da48e05b0c573dada1dd8b762864d10f4dc399713
+COMPONENT_ARCHIVE_URL= \
+	https://www.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	http://e2fsprogs.sourceforge.net/
+COMPONENT_FMRI=		sysutils/e2fsprogs
+COMPONENT_CLASSIFICATION= Applications/System Utilities
+COMPONENT_LICENSE=	gnu-gpl-v2 AND gnu-lgpl-v2 AND modified-bsd AND mit
+
```

Comment:
Did you invent this notation of mixed licenses or did you see it elsewhere? I only have seen a different notation yet.

Reply:
I think you misunderstood my comment about license notation. What I meant was we are already using a notation for multiple licenses like this:
COMPONENT_LICENSE=	GPLv2,GPLv3,LGPLv2,LGPLv2.1
You had used an uncommon variant.